src/backend/controllers/emailControllers.ts

```diff
@@ -61,7 +64,7 @@ export async function sendBookingConfirmationEmail(booking: ApiBooking) {
   }
   const { startDay, startTime, endTime } = getBookingTimeComponents(booking);
 
-  return sendEmailWithDynamicTemplate({
+  return sendEmailWithDynamicTemplate<BookingDynamicTemplate>({
     to: booking.email,
     from: {
       name: 'Naisten Linja Booking Notifcation',
```

Comment:
Let's fix this typo although they might want the sender name to be translated too?